### DISCUSSION TITLE
Fejl ved push til branch

### DISCUSSION THREAD

**Discussion by steffenjimclausen** [devcronberg/lp-cs-202403#4] [General]
Jeg får nedenstående fejl når jeg forsøger at pushe en ændring til branch.
Hvad gør jeg forkert ?

Pushing feature/consolehelper-input
Remote: Permission to devcronberg/lp-cs-202403.git denied to steffenjimclausen.
Error encountered while pushing to the remote repository: Git failed with a fatal error.
Git failed with a fatal error.
unable to access 'https://github.com/devcronberg/lp-cs-202403.git/': The requested URL returned error: 403

Failed to push to the remote repository. See the Output window for more details.

**Comment by mcronberg** [comment]
Jeg kigger lige i aften

Mvh Michell

ons. 6. mar. 2024 10.41 skrev steffenjimclausen ***@***.***>:

**Comment by devcronberg** [comment]
Har du clonet dit eller mit repo? Du skal clone dit, commit'e, pushe til dig - og så en PR mod mit. Giver det mening

**Comment by steffenjimclausen** [comment]
Hej Michell

Tak for din mail .

Skal jeg ikke clone dit repo for at få koden ned ? Og så lave en branch ?
________________________________
Fra: devcronberg ***@***.***>
Sendt: Wednesday, March 6, 2024 8:35:14 PM
Til: devcronberg/lp-cs-202403 ***@***.***>
Cc: Clausen, Steffen Jim ***@***.***>; Author ***@***.***>
Emne: Re: [devcronberg/lp-cs-202403] Fejl ved push til branch (Discussion #4)


Har du clonet dit eller mit repo? Du skal clone dit, commit'e, pushe til dig - og så en PR mod mit. Giver det mening

—
Reply to this email directly, view it on GitHub<https://github.com/devcronberg/lp-cs-202403/discussions/4#discussioncomment-8698282>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BEAMZQFPVK4W73QZCHFLEM3YW5VXFAVCNFSM6AAAAABEIWLBKGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DMOJYGI4DE>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

Nordic Sugar A/S
Edvard Thomsens Vej 10, 7.sal, 2300 København S, Danmark
CVR No: 29781834


Denne e-mail kan indeholde fortrolige oplysninger. Hvis du ikke er den tiltænkte modtager
men modtager denne e-mail ved en fejltagelse, bedes du give os besked og slette e-mailen
med alle vedhæftede filer. Det er ikke tilladt at offentliggøre, kopiere eller bruge
e-mailen eller at distribuere e-mailen eller dens bilag til andre personer.
Vi takker for din hjælp.

This e-mail may contain confidential information. If you are not the intended recipient
and you receive this e-mail by mistake, please notify us and delete the e-mail with all attachments.
Any unauthorized use, disclosure, copying or distribution of such e-mail or its attachments
to any other person is not allowed.
Thank you for your assistance.

Du finder her vores privatlivspolitik i henhold til artikel 13, 14 og 21 i den generelle forordning om databeskyttelse (GDPR):
You can find our privacy notices according to Art 13, 14 and 21 GDPR here:
Privatlivspolitik<https://www.nordzucker.com/en/privacy-notice>

**Comment by mcronberg** [comment]
Nope - du får ikke lov at pushe til et andet repo.

Fork til dig selv, clone det, branch ud, ret, commit, push - og så PR mod
mit. Du behøver i virkeligheden ikke lave en branch på dit eget - det er
måske for at have det originale og så din rettelse

**Reply by steffenjimclausen** [reply]
Hej Michell

Så har jeg forsøgt, at lave et pull request.

Jeg var ikke med på at man skulle forke til sit eget repo først. Det hjalp ;).

Tak for hjælpen.
